stdlib/LinearAlgebra/src/svd.jl

```diff
@@ -54,14 +54,15 @@ function SVD{T}(U::AbstractArray, S::AbstractVector{Tr}, Vt::AbstractArray) wher
         convert(AbstractArray{T}, Vt))
 end
 
+
 # iteration for destructuring into components
 Base.iterate(S::SVD) = (S.U, Val(:S))
 Base.iterate(S::SVD, ::Val{:S}) = (S.S, Val(:V))
 Base.iterate(S::SVD, ::Val{:V}) = (S.V, Val(:done))
 Base.iterate(S::SVD, ::Val{:done}) = nothing
 
 """
-    svd!(A; full::Bool = false) -> SVD
+    svd!(A; full::Bool = false, alg::Algorithm = DivideAndConquer()) -> SVD
```

Comment:
I would consider a 
```julia
default_svd_algorithm(A) = DivideAndConquer()
```
then
```julia
svd!(A; full::Bool = false, alg::Algorithm = default_svd_algorithm(A))
```
which would keep things flexible (cf `Sort.defalg` in `Base`).